src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int BASEINT = 2;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] map;
+
+    public MyHashMap() {
+        map = new Node[INITIAL_CAPACITY];
+        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    }
+
+    private static class Node<K,V> {
+        private final K key;
+        private V value;
+        private MyHashMap.Node<K, V> next;
+
+        public Node(K key, V value, MyHashMap.Node<K,V> next) {
```

Comment:
it should work like this
```suggestion
        private Node<K, V> next;

        public Node(K key, V value, Node<K,V> next) {
```